src/fuzz/fuzz-main.c

```diff
@@ -22,7 +22,9 @@ int main(int argc, char **argv) {
         size_t size;
         char *name;
 
-        test_setup_logging(LOG_DEBUG);
+        /* We don't want to fill the logs with messages about parse errors.
+         * Disable most logging if not running standalone */
+        test_setup_logging(LOG_CRIT);
 
```

Comment:
This doesn't seem right. We disable logging in each LLVMFuzzerTestOneInput() call to disable logging when fuzzers are executed from libFuzzer or such. With this patch, logging level is only set in fuzz-main.c:main(). This code path is not used when fuzzing.

Also, the effective log level when running a fuzzer from the command line is changed to LOG_CRIT, which also is wrong. We still want LOG_DEBUG.

I'd say that this is a good example of why commit messages are important:
>  fuzz: set the default log level in fuzz-main.c 

This doesn't tell me what the *intent* was:
- just a cleanup without change of logic (in which case I can describe how the fuzzing code path looks and why the log level will not be set appropriately)
- or are the levels changed intentionally (and if yes, then why?).

Reply:
Oh, I forgot that fuzz-main.c is only used on a certain case, and not always. Will drop this change. Thanks.

Reply:
Setting needs-rework flag for now.